client/src/LanguageSelection.js

```diff
@@ -1,19 +1,58 @@
-const LanguageSelection = () => {
-    return (
-        <div className="ui container">
-            <div className="content">
-                <h3 className="ui green header">Language Selection</h3>
-            </div>
-            <div className="content">
-                <select className="ui fluid dropdown">
-                    <option value="">Language</option>
-                    <option value="2">C</option>
-                    <option value="1">C++</option>
-                    <option value="0">Rust</option>
-                </select>
+import React from "react";
+
+class LanguageSelection extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            language: ""
```

Comment:
I like that idea. We can incorporate that change into our next week of work. Also, since I know you tested CodeMirror, can you see if C/C++ seem to work properly for you? It clearly changes from Rust and has the right setting in the code, but it just doesn't look like the example does. Is it a dark/light mode thing?

Reply:
if i had to guess it's probably something to do with the theme that i picked (at random for no good reason). we can test out some themes and see if others look better with C/C++ code this week.